src/service_client/assisted_service_api.py

```diff
@@ -76,7 +76,7 @@ def refresh_api_key(config: Configuration) -> None:
             }
 
             log.info("Refreshing API key")
-            response = requests.post(os.environ.get("SSO_URL"), data=params)
+            response = requests.post(os.environ["SSO_URL"], data=params)
```

Comment:
to fail fast in case of missing env var

Reply:
Might we do a try/except here so that we may provide a custom error message? 
Something like "The environment variable SSO_URL is mandatory but was not supplied".